.travis.yml

```diff
@@ -30,7 +30,12 @@ script:
   - black --check webviz_subsurface tests setup.py
   - pylint webviz_subsurface tests setup.py
   - bandit -r -c ./bandit.yml webviz_subsurface tests setup.py
-  - pytest tests
+  - webviz certificate
+  - git clone --depth=1 https://github.com/equinor/webviz-subsurface-testdata.git
+  - pip install git+https://github.com/equinor/libecl
+  - pip install git+https://github.com/equinor/fmu-ensemble
+  - sed -i "s&./webviz-subsurface-testdata& $PWD/webviz-subsurface-testdata&g" examples/reek_example.yaml
```

Comment:
- The . in `./webviz...`needs to be escaped with leading \
- And then the space before `$PWD`